hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: olm-artifacts-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  value: staging
+- name: IMAGE_TAG
+  value: latest
+
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: aws-account-operator-catalog
+  spec:
+    sourceType: grpc
+    image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
+    displayName: aws-account-operator Registry
+    publisher: SRE 
+
+- apiVersion: operators.coreos.com/v1alpha2
+  kind: OperatorGroup
+  metadata:
+    name: aws-account-operator-og
+  spec:
+    targetNamespaces:
+    - hive
```

Comment:
Should this be the same namespace as `hive`? maybe `aws-account-operator`?

Reply:
Fixed!